src/java.desktop/share/classes/javax/swing/JPasswordField.java

```diff
@@ -60,6 +60,14 @@
  * If an application needs the input methods support, please use the
  * inherited method, <code>enableInputMethods(true)</code>.
  * <p>
+ * <strong>Warning:</strong> The text entered in {@code JPasswordField} displays
+ * something that was typed and does not show the original characters. This
```

Comment:
"something that was typed" ? That reads to me as if it displays the characters you typed which isn't what you mean

the whole sentence is weird

Let's change this to
Although the text displayed in {@code JPasswordField} does not show the actual password characters typed,  this does not prevent ...

Reply:
The "something was typed" text is used already in the description of the JPasswordField, in the first sentence.

Not sure that the word "text" in the "Although the text displayed" is the correct one. It is not necessary a "text" but "something"

Reply:
The previous text you refer to is presumably
"where the view indicates  something was typed,"

that is fine. Your text is trying to say something else AND you added the word that which means you are referring to the specific thing .. 

So you can't just copy and edit that.

And "text" in my version is fine.

Reply:
So in case that wasn't clear, please change it to ny version

Reply:
Another way of phrasing it would be 

The text displayed in {@code JPasswordField} shows you that something was typed but does not show the original characters.

Note "that something" not "something that" and the rest of the rephrasing matters too.

Reply:
I do not like the phrase "Although the text displayed in", the "text" is not displayed in the JPasswordField per se, please suggest something else.

Reply:
"The text entered in {@code JPasswordField} displays that something was typed and does not show the original characters"
?

Reply:
change "and" to "but" and that will be good enough

"The text entered in {@code JPasswordField} displays that something was typed but does not show the original characters"